portal-impl/src/com/liferay/portal/util/PortalImpl.java

```diff
@@ -8100,6 +8095,22 @@ private Map<Locale, String> _getAlternateURLs(
 			Set<Locale> availableLocales)
 		throws PortalException {
 
+		PortalPreferences portalPreferences =
+			PortalPreferencesLocalServiceUtil.fetchPortalPreferences(
+				themeDisplay.getCompanyId(),
+				PortletKeys.PREFS_OWNER_TYPE_COMPANY);
+
+		com.liferay.portal.kernel.portlet.PortalPreferences newPortalPreferences =
+			PortalPreferenceValueLocalServiceUtil.getPortalPreferences(
+				portalPreferences, false);
+
+		if (Validator.isNotNull(newPortalPreferences.getValue(
+			null, "localePrependFriendlyURLStyle"))) {
+				PropsValues.LOCALE_PREPEND_FRIENDLY_URL_STYLE =
+					GetterUtil.getInteger(newPortalPreferences.getValue(
+						null, "localePrependFriendlyURLStyle"));
+		}
+
```

Comment:
@alberttgomes, please rebase interactively, edit this commit, run SF and commit again with `--amend`